public/app/features/templating/template_srv.ts

```diff
@@ -225,34 +230,57 @@ export class TemplateSrv {
     return values;
   }
 
+  getFieldAccessor(fieldPath: string) {
+    const accessor = this.fieldAccessorCache[fieldPath];
+    if (accessor) {
+      return accessor;
+    }
+
+    return (this.fieldAccessorCache[fieldPath] = _.property(fieldPath));
```

Comment:
Not a fan of such returns. What about changing this to:

```suggestion
   this.fieldAccessorCache[fieldPath] = _.property(fieldPath)
    return this.fieldAccessorCache[fieldPath];
```